src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -358,10 +363,28 @@ private void LogAuthResult(AuthenticationResult? authenticationResult)
                 throw new ArgumentException(IDWebErrorMessage.ClientCredentialTenantShouldBeTenanted, nameof(tenant));
             }
 
+            // If using managed identity 
+            if (tokenAcquisitionOptions != null && tokenAcquisitionOptions.ManagedIdentity != null)
+            {
+                try
+                {
+                    IManagedIdentityApplication managedIdApp = await GetOrBuildManagedIdentityApplication(
```

Comment:
>IManagedIdentityApplication [](http://example.com/codeflow?start=20&length=27)

try to use var instead to save an allocation

Reply:
I think var just helps make the code more readable, it doesn't add any perf penalty. The compiler replaces "var" with the type at compile (build) time.

https://stackoverflow.com/questions/41479/use-of-var-keyword-in-c-sharp

Whatever you decide on this, there are settings in VS that help enforce it, see 

![image](https://github.com/AzureAD/microsoft-identity-web/assets/12273384/8fd6d35f-5bd7-4cfb-bd66-eed6e869391e)

In MSAL we don't enforce this (we should) but the general opinion is that if the type is obvious, use `var` (e.g. `var x = new Account()`), else use the type.

Reply:
I mentioned in a different comment but [here are the docs for these options](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0007-ide0008), I agree we should align on one set as a team so it is unambiguous what to use and when.

Reply:
I have no preference, @JoshLozensky can decide.